buildenv/jenkins/variables/defaults.yml

```diff
@@ -163,11 +163,11 @@ boot_jdk_default:
       17: '17'
       19: '18'
       20: '18'
-      next: '18'
+      next: '20'
     dir_strip:
       all: '1'
     url:
-      all: "https://api.adoptopenjdk.net/v3/binary/latest/${bootJDKVersion}/ga/${os}/${arch}/jdk/openj9/normal/adoptopenjdk?project=jdk"
+      all: 'https://api.adoptium.net/v3/binary/latest/${bootJDKVersion}/ga/${os}/${arch}/jdk/hotspot/normal/eclipse?project=jdk'
```

Comment:
We should only be using hotspot for `next` and openj9 for the others.

Reply:
I tried using "openj9" for 17, I got a 404 error.

Reply:
Perhaps @AdamBrousseau knows, I assume it used to work. It's only needed when we have a new machine that doesn't already have the binary. We could leave versions earlier than `next` as-is for now in order to deliver the jdk21 fix.

Reply:
Adam explained that while adoptopenjdk.net redirects to adoptium.net, the redirection doesn't apply to api.adoptopenjdk.net. I'll update this accordingly.

Reply:
Updated.